algorithm/data_structure/chunkifyArray.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @param { Array[*] } arr 
+ * @param { Integer } splitInto 
+ * @returns { Array[Array[*]] }
+ */
+function chunkifyArray(arr, splitInto) {
+    if(!Number.isInteger(splitInto)) throw new Error("Array chunk must be divided by an integer number!");
```

Comment:
Bagian sebelah sini perlu di wrap pake kurung kurawal buat benerin issue [codacy](https://app.codacy.com/gh/bellshade/Javascript/pullRequest?prid=8405540)nya.

```suggestion
    if (!Number.isInteger(splitInto)) {
        throw new Error("Array chunk must be divided by an integer number!");
    }
```